base_multi_image/hooks.py

```diff
@@ -29,9 +29,7 @@ def post_init_hook_for_submodules(cr, model, field):
     with cr.savepoint():
         column_exists = table_has_column(cr, table, field)
         if column_exists:
-            cr.execute(
-                "SELECT id FROM %(table)s WHERE %(field)s IS NOT NULL"
-            )  # pylint: disable=sql-injection
+            cr.execute("SELECT id FROM %(table)s WHERE %(field)s IS NOT NULL")  # pylint: disable=sql-injection
```

Comment:
better way would be to use explicit AsIs function, so you would not need to disable pylint.

Reply:
@oerp-odoo it's leftover. No need this `pylint` disable anymore, removed it.